docs/Functional.md

```diff
@@ -1,51 +1,121 @@
+- The service will take into consideration the [Equation of time](https://en.wikipedia.org/wiki/Equation_of_time).
 
 ### Non-functional requirements
 
 - The service will be able to handle 10k requests per minute.
 - The service will not call an external API.
+- The service will be scalable thanks to kubernetes
+
+## Personae
+
+<img src="./thispersondoesnotexist/dev2.jpg" alt="drawing" width="200"/><br>
+Name: Marc<br>
+Age: 22<br>
+Education: Master's degree in computer science<br>
+Background: Student<br>
+Skills: Good knowledge of computer science<br>
+Geography: France<br>
+Platform: Web Desktop<br>
+Goal: Find out when Santa will arrive<br>
+Motivation: I want to know why this website exists<br>
+Pain points or frustrations: I don't like to sign up to a service<br>
+
+<img src="./thispersondoesnotexist/kid.jpg" alt="drawing" width="200"/><br>
+Name: Lisa<br>
+Age: 8<br>
+Education: None<br>
+Background: Child<br>
+Skills: Knows of the existance of Santa<br>
+Geography: UK<br>
+Platform: Web Ipad<br>
+Goal: Find out when Santa will arrive<br>
+Motivation: Embush Santa to prank him<br>
+Pain points or frustrations: I don't like hard interfaces<br>
+
+<img src="./thispersondoesnotexist/old.jpg" alt="drawing" width="200"/><br>
+Name: James<br>
+Age: 65<br>
+Education: Right's degree<br>
+Background: Retired<br>
+Skills: Knows how to navigate a web page as long as it's not too hard<br>
+Geography: USA<br>
+Platform: Web Desktop<br>
+Goal: Show to his grandsons when Santa will arrive<br>
+Motivation: I want to see the exitment in my grandsons' eyes<br>
+Pain points or frustrations: I can't navigate if the interface is not easy to use<br>
+
 
 ## Objectives.
-<!--Objectives are written by product designers in response to the requirements. They describe in a more specific way what the product will look like. Objectives may describe architectures, protocols and standards to which the product will conform. Measurable objectives are those that set some criteria by which the end product can be judged. Measurability can be in terms of some index of customer satisfaction or in terms of capabilities and task times. Objectives must recognize time and resource constraints. The development schedule is often part or a corollary of the objectives.-->
 
 It will be a single web page with an input field. The user will be able to enter a postal address or a coordinate and the page will show the time for Santa to arrive on the page.
 The service will be able to scale depending on the number of requests per minute.
 It will be supported by the majority of current browsers OSs and devices. 
 
 ## User documentation.
-<!--In general, all of the preceding documents (except the logic specification) are used as source material for the technical manuals and online information, such as help pages, that are prepared for the product's users.-->
 
 Write your adress in the field. And press enter.
```

Comment:
What if i don't have Internet, Does it still works ? 
What about on Phone, I don't have enter Button. 

What if a accept GPS position is accepted ? Do i still have to enter an address and press enter ?

Reply:
If internet disconnected right before I enter my address

Reply:
Is the website able to work only with cached data?